app/services/signature-service.js

```diff
@@ -66,9 +66,6 @@ export default class SignatureService extends Service {
     // Prepare sign flow: create preparation activity and send to SH
     const response = await uploadPiecesToSigninghub(signFlows);
     if (!response.ok) {
-      for (let signFlow of signFlows) {
-        await this.removeSignFlow(signFlow, true);
-      }
```

Comment:
forgot to mention, when we set the status/creator in the frontend and reset it in the backend, we will need to refresh those relations here, when the response is not ok or the store will not be up to date
```
     for (let signFlow of signFlows) {
        await signFlow.belongsTo('status').reload();
         await signFlow.belongsTo('creator').reload();
      }
```